bin/play

```diff
@@ -10,4 +10,4 @@ game = Game.new
 game.welcome_instructions
 game.get_game_type
 game.play_game
-game.show_end_of_game_message
+game.outcome
```

Comment:
Can you make it obvious that outcome has a side-effect? show_outcome maybe?

Reply:
#31